test/integration/controlplane/synthetic_controlplane_test.go

```diff
@@ -22,6 +22,9 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	authauthenticator "k8s.io/apiserver/pkg/authentication/authenticator"
+	authenticatorunion "k8s.io/apiserver/pkg/authentication/request/union"
+	"k8s.io/apiserver/pkg/authorization/authorizerfactory"
```

Comment:
nit: group these imports with the k8s imports below

Reply:
done